README.md

```diff
@@ -127,6 +127,17 @@ qemu-system-x86_64 -nographic -kernel path/to/kernel -initrd /tmp/initramfs.linu
 Note that you do not have to build a special kernel on your own, it is
 sufficient to use an existing one. Usually you can find one in `/boot`.
 
+For framebuffer support, append a VESA mode via the `vga` kernel parameter:
```

Comment:
for framebuffer support on ... Linux?

Reply:
Yes, that will get you `/dev/fb0` via the vesafb driver on the Linux system we run within QEMU, e.g. with a kernel from Ubuntu which has the config option.

Reply:
See the "Frame buffer Devices" section:
https://kernel.ubuntu.com/~kernel-ppa/config/eoan/linux/5.2.0-9.10/amd64-config.flavour.generic

Reply:
I should be more explicit, thanks. :)